elliptic-curve/src/lib.rs

```diff
@@ -62,8 +62,9 @@ use generic_array::{typenum::Unsigned, ArrayLength, GenericArray};
 use rand_core::{CryptoRng, RngCore};
 use subtle::{ConditionallySelectable, ConstantTimeEq, CtOption};
 
-/// Byte array containing a serialized scalar value (i.e. an integer)
-pub type ElementBytes<C> = GenericArray<u8, <C as Curve>::FieldSize>;
+/// Byte array containing a serialized field element, i.e. an element
+/// of the given curve's base or scalar fields.
+pub type FEBytes<C> = GenericArray<u8, <C as Curve>::FieldSize>;
```

Comment:
@str4d I've noticed the `ff` and `group` crates use a "Repr" naming convention here... do you think something like `FERepr` would be better?